code/modules/jobs/job_types/roguetown/adventurer/types/antag/hedgeknight.dm

```diff
@@ -18,11 +18,12 @@
 		STATKEY_SPD = 1,
 	)
 	subclass_skills = list(
-		/datum/skill/combat/polearms = SKILL_LEVEL_JOURNEYMAN,
+		/datum/skill/combat/polearms = SKILL_LEVEL_EXPERT,
```

Comment:
I don't think the HK should be getting expert in all weapon types. They're only given swords in their shop to begin with.